packages/next/src/utils.ts

```diff
@@ -2784,7 +2784,7 @@ interface EdgeFunctionInfoV2 extends BaseEdgeFunctionInfo {
 
 interface EdgeFunctionInfoV3 extends BaseEdgeFunctionInfo {
   matchers: EdgeFunctionMatcher[];
-  environments: Record<string, string>;
+  env: Record<string, string>;
```

Comment:
This appears to be a breaking change so we need to bump the manifest version and ensure we handle both `environments` and `env` right?

Reply:
I think this is fine because we have no stable version using that version package in production as far as I can tell!

Reply:
I see yea if it's not relied on for functionality in a proper release this is fine